cpp/include/cudf/aggregation.hpp

```diff
@@ -362,72 +375,55 @@ std::unique_ptr<Base> make_row_number_aggregation();
- *   1. venue:  (STRING) Location of the race event
- *   2. driver: (STRING) Name of the car driver (abbreviated to 3 characters)
- *   3. time:   (INT32)  Time taken to complete the circuit
+ * Produces the following average rank column:
+ * {   1,     2,   3.5,   3.5,     5,      1,   2.5,   2.5,     4,     5}
+ * (This corresponds to the following grouping and `driver` rows:)
+ * { "HAM", "LEC", "BOT", "NOR", "RIC",  "RIC", "NOR", "BOT", "LEC", "PER" }
+ *   <----------silverstone----------->|<-------------monza-------------->
  *
- * For the following presorted data:
+ * Produces the following min rank column:
+ * {   1,     2,     3,     3,     5,      1,     2,     2,     4,     5}
+ * (This corresponds to the following grouping and `driver` rows:)
+ * { "HAM", "LEC", "BOT", "NOR", "RIC",  "RIC", "NOR", "BOT", "LEC", "PER" }
+ *   <----------silverstone----------->|<-------------monza-------------->
  *
- *  [ //      venue,           driver,           time
- *    {   "silverstone",  "HAM" ("hamilton"),   15823},
- *    {   "silverstone",  "LEC" ("leclerc"),    15827},
- *    {   "silverstone",  "BOT" ("bottas"),     15834},  // <-- Tied for 3rd place.
- *    {   "silverstone",  "NOR" ("norris"),     15834},  // <-- Tied for 3rd place.
- *    {   "silverstone",  "RIC" ("ricciardo"),  15905},
- *    {      "monza",     "RIC" ("ricciardo"),  12154},
- *    {      "monza",     "NOR" ("norris"),     12156},  // <-- Tied for 2nd place.
- *    {      "monza",     "BOT" ("bottas"),     12156},  // <-- Tied for 2nd place.
- *    {      "monza",     "LEC" ("leclerc"),    12201},
- *    {      "monza",     "PER" ("perez"),      12203}
- *  ]
+ * Produces the following max rank column:
+ * {   1,     2,     4,     4,     5,      1,     3,     3,     4,     5}
+ * (This corresponds to the following grouping and `driver` rows:)
+ * { "HAM", "LEC", "BOT", "NOR", "RIC",  "RIC", "NOR", "BOT", "LEC", "PER" }
+ *   <----------silverstone----------->|<-------------monza-------------->
  *
- * A grouped dense rank aggregation scan with:
- *   groupby column      : venue
- *   input orderby column: time
  * Produces the following dense rank column:
  * {   1,     2,     3,     3,     4,      1,     2,     2,     3,     4}
  * (This corresponds to the following grouping and `driver` rows:)
  * { "HAM", "LEC", "BOT", "NOR", "RIC",  "RIC", "NOR", "BOT", "LEC", "PER" }
  *   <----------silverstone----------->|<-------------monza-------------->
  * @endcode
+ *
+ * @param method The ranking method used for tie breaking (same values).
+ * @param column_order The desired sort order for ranking
+ * @param null_handling  flag to include nulls during ranking. If nulls are not included,
+ * corresponding rank will be null.
+ * @param null_precedence The desired order of null compared to other elements for column
+ * @param percentage flag to convert ranks to percentage in range (0,1}
```

Comment:
> ranks to percentage in range (0,1}
 
Should this not be `in the range (0, 1]`? i.e. Excludes `0`, includes `1`?
(Sorry, I'm not familiar with `}` in this context.)

Reply:
This is a typo. Copied from old code. thanks.